inference-engine/src/cldnn_engine/ops/pad.cpp

```diff
@@ -27,8 +27,10 @@ static std::vector<int32_t> GetPermuteOrder(const ngraph::CoordinateDiff& ie_ord
     std::vector<int32_t> cldnn_order(ie_order.begin(), ie_order.end());
```

Comment:
The `ie_order` is empty in those tests and the segfault was caused by the `std::swap` invocation below the fixed code.